main/src/library/BigInt.flix

```diff
@@ -16,6 +16,8 @@
 
 namespace BigInt {
 
+    use ToString.toString;
```

Comment:
Actually, can you just have this use locally? or perhaps even better, just write `ToString.toString`?

Reply:
For `BigInt` specifically or all the files?

Reply:
All :)